pgxctx/logger.go

```
@@ -0,0 +1,86 @@
+		queryName = ctxdata.name
+	}
+
+	dur := time.Since(data.start)
+	if dur > slow {
+		reqID, ok := requestid.FromContext(ctx)
+		reqidstr := "<none>"
+		if ok {
+			reqidstr = reqID.String()
+		}
+
+		level.Warn(s.logger).Log(
```

golangci-lint (2.12.2)

[error] Error return value of `(github.com/go-kit/log.Logger).Log` is not checked

(errcheck)